README.rst

```diff
@@ -45,14 +45,12 @@ About
 The **nimi-python** repository generates Python bindings (Application Programming Interface) for interacting with the Modular Instrument drivers. Currently, the following drivers are supported:
 
 * NI-DCPower (Python module: nidcpower)
-* NI-Digital Pattern Driver (Python module: nidigital)
```

Comment:
@marcoskirsch, changes in this file is because I updated README without realizing that it's codegen'd. I'm looking into how to update this file the correct way.